README.md

```diff
@@ -1,125 +1,130 @@
+
+### docker-compose.test.watch.yaml
+This file is intended to be an override file for `docker-compose.test.yaml`.  The container will not exit following test run, instead it will watch for code changes in the application or tests and rerun on occurence.
+
+`docker-compose -p ffc-demo-payment-service-core-test -f docker-compose.yaml -f docker-compose.test.watch.yaml up`
+
+## Running the application
+The application is designed to run in containerised environments, using Docker Compose in development and Kubernetes in production.
+- A Helm chart is provided for production deployments to Kubernetes.
+
+### Build container image
+Container images are built using Docker Compose, with the same images used to run the service with either Docker Compose or Kubernetes.
+
+By default, the start script will build (or rebuild) images so there will rarely be a need to build images manually. However, this can be achieved through the Docker Compose [build](https://docs.docker.com/compose/reference/build/) command:
+
+```
+# Build container images
+docker-compose build
+```
+
+### Start and stop the service
+Use Docker Compose to run service locally.
+
+```
+# Start the service in development
+docker-compose up
+```
+
+### docker-compose.override.yaml
+
+The default `docker-compose.yaml` and `docker-compose.override.yaml` provide the following features to aid local development:
+
+- map port 3007 from the host to the app container
+- bind-mount application code into the app container
+- run the application behind a file watcher, automatically reloading the app on change
+- run a database and message queue alongside the application
+
+Additional Docker Compose files are provided for scenarios such as linking to other running services and running automated tests.
+
+### docker-compose.link.yaml
+This will link to other FFC Demo services running locally.
+
+`docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up`
+
+### Deploy to Kubernetes
+
+For production deployments, a helm chart is included in the `.\helm` folder. This service connects to an AMQP 1.0 message broker, using credentials defined in [values.yaml](./helm/ffc-demo-payment-service-core/values.yaml), which must be made available prior to deployment.
+
+#### Accessing the pod
+By default, the service is not exposed via an endpoint within Kubernetes.
+
+Access may be granted by forwarding a local port to the deployed pod:
+
+```
+# Forward local port to the Kubernetes deployment
+kubectl port-forward --namespace=ffc-demo deployment/ffc-demo-payment-service-core 3007:3007
+```
+
+Once the port is forwarded, the service can be accessed and tested in the same way as described in the "Test the service" section above.
+
+#### Probes
+The service has both an Http readiness probe and an Http liveness probe configured to receive at the below end points.
+
+Readiness: `/healthy`
+Liveness: `/healthz`
+
+## License
```

Comment:
Should be Licence as noun.

Reply:
Corrected. Thanks, I never knew that until now.